multi_livechat/models/demo/mail_channel.py

```diff
@@ -9,4 +9,5 @@ class MailChannel(models.Model):
     channel_type = fields.Selection(
         selection_add=[("multi_livechat_echo_demo", "ECHO (Demo)")],
         ondelete={"multi_livechat_echo_demo": "cascade"},
+        readonly=False,
```

Comment:
I'm not sure that it's a good idea to switch channel type

Reply:
Yes, it's not a good idea. The attribute readonly I removed.